### ISSUE TITLE
edb-debugger-1.0.0-r2 : /.../ByteShiftArray.cpp:83:15: error: INT_MAX was not declared in this scope

### ISSUE THREAD

**Issue by Petross404** [eteran/edb-debugger#736]
There is a bug at bugs.gentoo about a compilation error with CGG-9.2. It seems there is a missing include file?

```
/var/tmp/portage/dev-util/edb-debugger-1.0.0-r2/work/edb-debugger-1.0.0/src/ByteShiftArray.cpp:83:15: error: ‘INT_MAX’ was not declared in this scope
   83 |  Q_ASSERT(i < INT_MAX);
      |               ^~~~~~~
/var/tmp/portage/dev-util/edb-debugger-1.0.0-r2/work/edb-debugger-1.0.0/src/ByteShiftArray.cpp:20:1: note: ‘INT_MAX’ is defined in header ‘<climits>’; did you forget to ‘#include <climits>’?
```

(https://bugs.gentoo.org/701112)

**Comment by eteran** [comment]
This was a bug, but has been fixed since the 1.0.0 release.

I would advise Gentoo (My favorite Distro!) to:

1. Have the ebuild patch the 1.0.0 sources because it's a trivial one-liner include.
2. Add an ebuild for version 1.1.0 :-).

I'm closing because this issue doesn't effect the latest version or master.

**Comment by Petross404** [comment]
Thanks for the help.
